docs/content/docs/user-docs/resource-crud.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "Resource Creation"
+description: "Create and Delete an S3 bucket"
+lead: "Create and Delete an S3 bucket using ACK controller"
+draft: false
+menu:
+  docs:
+    parent: "installing"
+weight: 30
+toc: true
+---
+
+## Create an S3 bucket
+
+```bash
+
+export BUCKET_NAME=my-ack-s3-bucket-$AWS_ACCOUNT_ID
```

Comment:
I know we set `AWS_ACCOUNT_ID` earlier, but I wouldn't count on it still being exported. I'd add `export AWS_ACCOUNT_ID=$(aws sts get-caller-identity --query "Account" --output text)` again, anyway.